wpa_supplicant/wpa_supplicant.c

```diff
@@ -905,6 +905,27 @@ void wpa_supplicant_reinit_autoscan(struct wpa_supplicant *wpa_s)
 	}
 }
 
+// TODO: This WAR is needed as we always lose the first frame after association (DHCP),
```

Comment:
Is this a final workaround or there is some kind of different remedy pending for that?

Reply:
Yes, there is a bug against sheliak, but as its a ROM code, will take time, so, this is the final WAR for now.